Added iPad support and some improvements

kronik opened the pull request:
1. Added becomeFirstResponder / resignFirstResponder methods
2. Fixed text field size for iPad
3. Some improvements

kronik committed:
1. Added becomeFirstResponder / resignFirstResponder methods
2. Fixed text field size for iPad
3. Some improvements

brandonhamilton commented:
Thanks for this !
Does this work properly when changing orientation (e.g. to landscape) ? - In my testing the field seems to disappear off screen when the keyboard is opened in landscape mode.

kronik committed:
Fixed resizing on device rotation

kronik commented:
Hi,

Yes, my bad… In my app I don't change screen orientation. Anyway I've fixed this issue and simplified code a bit. But still can't fix one small glitch:
1. Start app in portrait mode
2. Enter several words to get 3-4 lines of text
3. Change device orientation to landscape. Text view resizes properly.
4. Change device orientation back to portrait. Text view resizes properly but text formatting is broken: it looks like in text every space was replaced with '\n' char.

Could you take a look on this issue? https://github.com/kronik/inputtoolbar.git

Thank you.  

## 

Best Regards,
Dmitry Klimkin

On Friday, 5 April, 2013 at 1:32 AM, Brandon Hamilton wrote:

> Thanks for this !
> Does this work properly when changing orientation (e.g. to landscape) ? - In my testing the field seems to disappear off screen when the keyboard is opened in landscape mode.  
> 
> —
> Reply to this email directly or view it on GitHub (https://github.com/brandonhamilton/inputtoolbar/pull/15#issuecomment-15911750).

brandonhamilton commented:
Sorry for the long delay in responding. There seems to still also be another issue caused by orientation change: start in portrait mode, type some text and then push the button to hide the keyboard. Once the keyboard is hidden, when changing orientation the toolbar placement is out (disappears in landscape mode).

kronik commented:
Hi, yes seems you're right. I used this component in portrait mode only. But now don't have enough time to fix the issue you mentioned.  

## 

Best Regards,
Dmitry Klimkin

Senior Mobile Developer
Bubbly

On Wednesday, 24 July, 2013 at 5:18 PM, Brandon Hamilton wrote:

> Sorry for the long delay in responding. There seems to still also be another issue caused by orientation change: start in portrait mode, type some text and then push the button to hide the keyboard. Once the keyboard is hidden, when changing orientation the toolbar placement is out (disappears in landscape mode).
> 
> —
> Reply to this email directly or view it on GitHub (https://github.com/brandonhamilton/inputtoolbar/pull/15#issuecomment-21472834).

brbgyn commented:
Have anyone solved this issue ?

brbgyn commented:
Havind kind of same issue here:
1. Start app in portrait mode
2. Enter several words to get a full line of text
3. Change device orientation to landscape. Text view resizes properly.
4. Change device orientation back to portrait. Text view resizes properly but text viewing gets cutted, as of touchable area that would activate the keyboard.
